libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/InsightDrillConfigPanel.tsx

```diff
@@ -113,6 +113,23 @@ export const InsightDrillConfigPanel: React.FunctionComponent<IDrillConfigPanelP
     const dispatch = useDashboardDispatch();
     const mergedItems = mergeDrillConfigItems(drillItems, incompleteItems);
 
+    const availableDrillTargetMeasures = configItems?.availableDrillTargets?.measures?.filter(
+        (measure) =>
+            mergedItems.find(
```

Comment:
You could use `!mergedItems.some(...)` instead of the `mergedItems.find(...) === undefined`, same for the next expression.